app/lib/configuration.js

```diff
@@ -20,6 +20,7 @@ class Configuration {
     this.theme = ''
     this.hotkey = ''
     this.debug = false
+    this.height = 700
```

Comment:
I intentionally set it to 700, since the purpose of this is to test if we can increase the window height, remember?